### PR TITLE
[WIP] Create, List, Delete polls

### PR DESCRIPTION
**Pull request by RafalWilinski**
https://monosnap.com/file/5NFaN4RXj9moNlUmn60NTdW7ydzkUf

### CODE REVIEW

**Comment by jacekelgda** [comment]
@RafalWilinski could you do a rebase from upstream master ? that way we will have a preview as well.

**Comment by RafalWilinski** [comment]
I've rebased PR but I can't see the deployment. Should I reopen this PR?

**Comment by jacekelgda** [comment]
No, that’s ok. It might be related to the fact that is fork. Don’t worry I will investigate this.

File: src/components/AddPoll.vue [hunk 1]
```diff
@@ -1,17 +1,50 @@
...
+      </el-form-item>
+       <el-form-item>
+        <el-input v-model="poll.second.value" placeholder="Second Option"></el-input>
+      </el-form-item>
+      <el-form-item>
+        <el-button type="primary" @click="onSubmit">Add Poll</el-button>
+      </el-form-item>
+    </el-form>
+    <ul class="polls" v-loading="isLoading" :data="polls">
       <li v-for="poll in polls">
```
Comment (mmiszy) [inline src/components/AddPoll.vue]: Seems like `v-bind:key` is missing

File: src/components/AddPoll.vue [hunk 2]
```diff
@@ -25,27 +58,74 @@ export default {
...
+          votes: 0,
+        },
+      },
     };
   },
   firebase: {
-    polls,
+    polls: {
+      source: polls,
+      readyCallback: function () { // eslint-disable-line
```
Comment (mmiszy) [inline src/components/AddPoll.vue]: let's change this to `readyCallback() {`

**Comment by jacekelgda** [comment]
👍

**Pull request merged**